ci/lint-commit-msg.py

```diff
@@ -95,7 +95,8 @@ def no_merge(body):
 
 @lint_rule("Subject line should be shorter than 72 characters")
 def line_too_long(body):
-	return len(body[0]) <= 72
+	revert = re.search(r"^Revert \"(.*)\"", body[0])
```

Comment:
Regex seems a bit heavy handed for this. Why not use [`startswith()`](https://docs.python.org/3/library/stdtypes.html#str.startswith) ?

Reply:
Well I wanted specifically the quotes too. It's just what the other search above it does.